mlflow/tracking/fluent.py

```diff
@@ -2025,6 +2025,36 @@ def search_logged_models(
         )
 
 
+def log_inputs(models: Optional[List[ModelInput]] = None):
```

Comment:
Instead of creating a new API for this, can we extend the existing `log_input` API?